ratelimit.toml

```diff
@@ -0,0 +1,47 @@
+[global]
```

Comment:
Did you add that file on purpose to the PR?

Reply:
Nope